astroid/tests/unittest_brain.py

```diff
@@ -908,5 +908,27 @@ def test_inferred_successfully(self):
         self.assertEqual(elems, [1, 2])
 
 
+class SubprocessTest(unittest.TestCase):
+    """Test subprocess brain"""
+    # TODO Add more tests so that we can some day
+    # Remove this brain when all the tests work without the brain
+    @unittest.skipIf(sys.version_info < (3, 3),
+                     reason="Python 2.7 subprocess doesnt have args")
+    def test_subprocess_args(self):
+        """Make sure the args attribute exists for Popen
+
+        Test for https://github.com/PyCQA/pylint/issues/1860"""
+        name = astroid.extract_node("""
+        import subprocess
+        p = subprocess.Popen(['ls'])
+        p #@
+        """)
+        [inst] = name.inferred()
+        try:
```

Comment:
This can be written as:
```
with self.assertRaises(Exception):
   inst.getattr(...)
```

Reply:
I actually don't want to assert that the test raises anything. I just want to make sure that the test fails properly on the occasion that it does fail: (F) instead of (E)

Reply:
Ah, I see, I read it wrongly. In this case, I find it confusing, the `except` clause should be removed and just assert that the `args` value points to something that you expect.